src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/StorePal.Windows.Export.cs

```diff
@@ -109,20 +109,20 @@ private unsafe byte[] ExportPkcs12Core(Pkcs12ExportPbeParameters? exportParamete
             Interop.Crypt32.PKCS12_PBES2_EXPORT_PARAMS* exportParams = null;
             PbeParameters? reEncodeParameters = null;
 
+            char* PKCS12_PBES2_ALG_AES256_SHA256 = stackalloc char[] { 'A', 'E', 'S', '2', '5', '6', '-', 'S', 'H', 'A', '2', '5', '6', '\0' };
+            Interop.Crypt32.PKCS12_PBES2_EXPORT_PARAMS specifiedParams = new()
+            {
+                dwSize = (uint)Marshal.SizeOf<Interop.Crypt32.PKCS12_PBES2_EXPORT_PARAMS>(),
```

Comment:
```suggestion
                dwSize = (uint)sizeof(Interop.Crypt32.PKCS12_PBES2_EXPORT_PARAMS),
```
Given that this code takes unmanaged pointer address of the structure without any marshalling, it is more correct and a lot faster to use `sizeof` instead of `Marshal.SizeOf`.

From https://learn.microsoft.com/en-us/dotnet/standard/native-interop/best-practices: If the struct is blittable, use sizeof() instead of Marshal.SizeOf<MyStruct>()

Reply:
Other recently introduced Marshal.SizeOf under System.Security.Cryptography can use the same treatment:
```
src\libraries\System.Security.Cryptography\src\System\Security\Cryptography\X509Certificates\ChainPal.Windows.BuildChain.cs(39):chainPara.cbSize = Marshal.SizeOf<Interop.Crypt32.CERT_CHAIN_PARA>();
src\libraries\System.Security.Cryptography\src\System\Security\Cryptography\X509Certificates\ChainPal.Windows.BuildChain.cs(94):customChainEngine.cbSize = Marshal.SizeOf<Interop.Crypt32.CERT_CHAIN_ENGINE_CONFIG>();
src\libraries\common\src\System\Security\Cryptography\MLKem.Windows.cs(29):int blobHeaderSize = Marshal.SizeOf<BCRYPT_MLKEM_KEY_BLOB>();
src\libraries\common\src\System\Security\Cryptography\PqcBlobHelpers.cs(233):int blobHeaderSize = Marshal.SizeOf<BCRYPT_MLKEM_KEY_BLOB>();
src\libraries\common\src\System\Security\Cryptography\ECCng.ImportExport.NamedCurve.cs(162):descPtr = Marshal.AllocHGlobal(Marshal.SizeOf(desc));
src\libraries\common\src\System\Security\Cryptography\ECCng.ImportExport.NamedCurve.cs(163):buffPtr = Marshal.AllocHGlobal(Marshal.SizeOf(buff));
```
The last two do not even need AllocHGlobal. It can be just a local.